src/containerUtils.js

```diff
@@ -0,0 +1,69 @@
+import invariant from 'invariant';
```

Comment:
I would suggest killing this Utils module and putting both functions that it exports into a separate module. `Utils` sucks :)